.travis.yml

```diff
@@ -4,6 +4,7 @@ osx_image: xcode9.3
 xcode_project: Alicerce.xcodeproj
 
 before_install:
+  - brew bundle
```

Comment:
don't you need install? perhaps for travis you don't, just asking 😊

Reply:
OSX Travis images have homebrew pre-installed :)